app/controllers/statistics_controller.rb

```diff
@@ -0,0 +1,17 @@
+class StatisticsController < ApplicationController
+  def count_metric
+    count_metric_snapshot = MetricSnapshot.where(code: params[:metric_code]).count
+  	total_configuration = MetricConfiguration.all.count
+
+    if(total_configuration == 0)
+      metric_percentage = 0
+    else
+      metric_percentage = (count_metric_snapshot.to_f / total_configuration.to_f) * 100
```

Comment:
It was my mistake actually. I didn't notice this.

But I'm still kind of convcerned that this if is present just because in the very first runs KalibroConfiguration may not have MetricConfigurations but in the other 99.99% of the time this if is useless.

What about a begin/rescue instead of the if? It is still kind of useless but if you rescue the `ZeroDivisionError` it will be clearer to the other programmers the reason behind the check.